.ruby-version

```diff
@@ -0,0 +1 @@
+ruby-2.7
```

Comment:
Can we use  an action-local configuration as opposed to having a global version for the whole repo?
```yaml
with:
  ruby-version: '2.7'
```